standards/mobile_app_standards.md

```diff
@@ -0,0 +1,135 @@
+Mobile Application Standards
+============================
+
+Scope
+-----
+
+These standards apply to mobile application in-house development and management.
+
+These standards are intended to cover all forms of mobile application: from fully "off-line capable" progressive web applications, through cross-platform apps to fully native apps.
+
+As implied, there are a number of ways of delivering mobile apps. These standards will cover which approach is best and under what circumstances, and aspects that are common to all or many of the approaches. As and when more detail is available on specific approaches, such as Flutter app development, or Progressive Web Apps, such approach-specific standards should form part of their own separate, detailed, documented standard, which, ideally, will simply be a reference to a recognised external standard for that specific technology.
```

Comment:
I don't think this needs to be stated here.

Reply:
I think we do need to state something along the lines that more detailed standards for specific techs will be written as and when they are necessary (and we know enough to actually write those standards!).